Add doc on downsampling and retention

rkuchan opened the pull request:
Originally: https://github.com/influxdb/influxdb.com/pull/286

rkuchan commented:
I still need to edit the links and take care of the sidebar for the new site.

beckettsean commented on content/influxdb/v0.9/guides/downsampling_and_retention.md:
```diff
@@ -0,0 +1,119 @@
+
+> **Clarifying** `default` **vs.** `DEFAULT`
+
+> `default`: The RP that InfluxDB automatically generates when you create a new database. It has an infinite duration and a replication factor set to the number of nodes in the cluster.
```
The name of the RP that InfluxDB automatically generates when you create a new database. It has an infinite duration and a replication factor set to the number of nodes in the cluster. It is initially the `DEFAULT` RP as well, but that can be altered.

beckettsean commented on content/influxdb/v0.9/guides/downsampling_and_retention.md:
```diff
@@ -0,0 +1,119 @@
+ * automatically aggregate the 10 second level data to 30 minute level data
+ * keep the 30 minute level data forever
+
+The following steps work with a fictional [database](/influxdb/v0.9/concepts/glossary/#database) called `food_data` and the [measurement](/influxdb/v0.9/concepts/glossary/#measurement) `orders`. `orders` has two [fields](/influxdb/v0.9/concepts/glossary/#field), `phone` and `website`, which store the number of orders that arrive through the relevant median every 10 seconds.
```
"relevant media" not "relevant median"

beckettsean commented:
minor notes, otherwise looks good to go!

beckettsean commented:
I do wonder if we need more about the RP syntax for writing to and selecting from non-DEFAULT RPs. Do the RP docs cover this in detail? I don't recall.

rkuchan commented:
Thanks, @beckettsean!

I made the requested edits and added a discussion about querying from and writing to an RP other than the `DEFAULT` to the `Working with RPs` section. I also added a note about the same topic to the `Create the CQ` section.

The Database Management doc doesn't talk about writing to and selecting from non`DEFAULT` RPs. There is a brief discussion in [Data Exploration](https://docs.influxdata.com/influxdb/v0.9/query_language/data_exploration/#the-basic-select-statement) and there's the somewhat related FAQ on [Frequently Encountered Issues](https://docs.influxdata.com/influxdb/v0.9/troubleshooting/frequently_encountered_issues/#missing-data-after-creating-a-new-default-retention-policy).

beckettsean commented:
+1
